docs/en/pid-eaa-data-model.rst

```diff
@@ -126,13 +133,13 @@ The ``verification`` claim is a JSON structure with all the following mandatory
       - **Description**
       - **Reference**
     * - **trust_framework**
-      - [NSD]. MUST be set to eidas
+      - [NSD]. For PID credential it MUST be set to ``eidas``. For (Q)EAA it MUST be set to ``eidas2``.
       - `[OID.IDA. Section 5.1] <https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html#section-5.1>`_
     * - **assurance_level**
-      - [NSD]. MUST be set to high
+      - [NSD]. MUST be set according to the LoA required. For PID credential it MUST be set to ``high``.
```

Comment:
```suggestion
      - [NSD]. MUST be set according to the required identity Level of Assurance identifier (LoA). For PID credential it MUST be set to ``high``.
```

we don't have a definition of LoA

Reply:
We should have LoA as a defined term. We use also in the sequence diagram where you proposed to add LoA in the previous PR...

Reply:
done